src/app/navbar/navbar.component.css

```diff
@@ -46,4 +50,21 @@ a:hover {
     background-color: white;
     color: black;
     transition: background-color 0.2s ease;
+}
+
+.collapsedButton {
+    border: solid;
+    border-color: white;
+    /* background-color: white; */
+    color: white;
+    transition: background-color 0.2s ease;
+}
+
+.collapsedButton:hover {
+    background-color: white;
+    color:#24292e;
+}
+
+#dropdown {
+    background-color: #24292e;
 }
```

Comment:
Added the CSS code for the differences in buttons between desktop and mobile mode. Also added a class specification for dividers in the navbar. (horizontal line for dividing mobile links)